RPIPs/RPIP-46.md

```diff
@@ -107,9 +113,17 @@ Some example concrete guidelines:
 - When there are large changes to the system (eg, Saturn 2 release), do note that some volatility is expected and should be considered when acting
 - If we are approaching the self-limits described in [RPIP-17](RPIP-17.md), the pDAO should act to limit one or both of rETH demand (via reducing RPL inflation spend on rETH demand and/or lower `reth_share`) or NO supply (via lower `no_share`). This would result in higher `surplus_share` (or lower RPL inflation).
 
+## Rationale
+UARS is meant to enable the protocol to listen to the market and act effectively across the entire protocol.
+A few details about the reasoning behind the spec:
+- We provide for future automated controller contracts, but do not create any at this time. This is partly because we are fairly naive to the market, and partly in the interest of time to market for the next upgrade.
+- The `node_operator_commission_share_council_adder` allows for much more rapidly tracking the market, especially when we first start UARS and may be quite far from the an appropriate `node_operator_commission_share + node_operator_commission_share_council_adder` value
+  - Due to the low maximum for the adder, the pDAO would need to act to enable much growth in `node_operator_commission_share + node_operator_commission_share_council_adder`. For example, if the adder is at 1%, the pDAO could vote to set it to 0% and add 1% to `node_operator_commission_share`. This active pDAO participation ensures that this setting tracks closely to the will of the pDAO. 
+- The [surplus revenue share vote](#surplus-revenue-share-vote) is intended to allow the main body of the tokenomics to move forward, while allowing more time to get information about our options here before choosing a path
+
 ## Security Considerations
-- `no_share_seal`s are intended to be used in a particular way, but the security council may misuse them
-  - This can be mitigated by limiting `no_share_seal_count`; eg, it can start high while the market is not well understood, but most of the time thereafter it can be at zero or one
+- `node_operator_commission_share_council_adder` is intended to be used in a particular way, but the security council may misuse it and increase it in other ways
```

Comment:
```suggestion
- `node_operator_commission_share_council_adder` is intended to be used in a particular way, but there is no on-chain mechanism to prevent the security council misusing it.
```

Reply:
Ah, now doesn't track with your point below.